src/app/project/board/url-template/url-template.component.ts

```diff
@@ -0,0 +1,59 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { IUrl } from '@dasch-swiss/dsp-js';
+
+@Component({
+    selector: 'app-url-template',
+    template: `
+        <div *ngIf="urls">
+            <div *ngIf="templateType == 'string'">
+                <p *ngFor="let str of urls" class="remove-top-margin"> {{ str }} </p>
+            </div>
+            <div *ngIf="templateType == 'IUrl'">
+                <div class="metadata-property">
+                    <div *ngIf="displayLabel" class="property-label display-inline-block">
+                        {{ label }}:
+                    </div>
+                    <div [ngClass]="{'display-inline-block add-left-margin': displayLabel}">
+                        <span *ngFor="let entry of urls" class="comma">
+                            <a *ngIf="entry.name" href="{{ entry.url }}" target="_blank"> {{ entry.name }} </a>
+                            <a *ngIf="!entry.name" href="{{ entry.url }}" target="_blank"> {{ entry.url }} </a>
+                        </span>
+                    </div>
+                </div>
+            </div>
+        </div>
+    `
+})
+export class UrlTemplateComponent implements OnInit {
+    @Input() urls: IUrl | IUrl[] | string[];
+
+    @Input() displayLabel ?: boolean = false;
+
+    @Input() isPerson ?: boolean = false;
```

Comment:
You can remove typings here because of initiation with the primitive values. Also safe operator `?` doesn't require space after property name.